apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/MainActivity.java

```diff
@@ -373,7 +373,7 @@ public void accept(Boolean crashed) {
             @Override
             public void accept(ErrorReport data) {
                 if (data != null) {
-                    Log.i(LOG_TAG, "Crashes.getLastSessionCrashReport().getThrowable()=", data.getThrowable());
+                    Log.i(LOG_TAG, "Crashes.getLastSessionCrashReport().getStackTrace()=" + data.getStackTrace());
```

Comment:
Since the API is new, it makes the jCenter build fail.
You need to use reflection here and add the usual release TODO comment.